src/main/java/org/sonarlint/intellij/editor/EditorDecorator.kt

```diff
@@ -107,6 +108,10 @@ open class EditorDecorator(private val project: Project) {
     createHighlight(hotspot.primaryLocation.range, hotspot.message)?.let { updateHighlights(listOf(it)) }
   }
 
+  open fun highlight(securityHotspot: LiveSecurityHotspot) {
```

Comment:
as there is nothing specific to security hotspots regarding highligthing, we could probably have a single method that takes a LiveFinding